elements/pfe-styles/src/pfe-type.scss

```diff
@@ -0,0 +1,65 @@
+@import "../../pfe-sass/pfe-sass";
+
+.pfe-headline-xxl {
+    @extend %pfe-headline-xxl;
+}
+
+.pfe-headline-xl {
+    @extend %pfe-headline-xl;
+}
+
+.pfe-generic h1,
```

Comment:
Instead of a pfe-generic class, I really like the idea of a pfe-generic component or pfe-text as you called it when you opened it previously.

Reply:
Or maybe in addition to?

Reply:
I was thinking that a text component would add a lot of markup cruft, vs a class:

```
<pfe-card class="pfe-generic">
    <h2>hello world</h2>
    <p>hey hey</p>
</pfe-card>
```
vs.
```
<pfe-card >
    <pfe-generic>
        <h2>hello world</h2>
        <p>hey hey</p>
    </pfe-generic>
</pfe-card>
```
multiplied times every chunk of text on a page.. could get clunky. I really liked the idea you mentioned this morning though, about baking in empty typographical vars into basic HTML tags, which spring to life once you add vars to the root of the page. Maybe we could do away with .pfe-generic in favor of that? This is stating the obvious but the classes like `.pfe-headline-xl` are still useful for type that is not inside a PFE component, so we can keep these too.